libs/coin-modules/coin-tezos/src/logic/craftTransaction.integ.test.ts

```diff
@@ -99,4 +99,35 @@ describe("Tezos Api", () => {
       ]),
     );
   });
+
+  it("should craft a send transaction from tz2 address with correct compressedreveal public key", async () => {
+    // Given: LAMA account from production environment
+    const tz2Address = "tz2F4XnSd1wjwWsthemvZQjoPER7NVSt35k3";
+    const expectedPublicKey = "sppk7but7h93Ws1XhAPvdBcttVmoBDGHxdpaU8dPy5549f3eLJFAjag";
+
+    // When: craft a send transaction with the public key
+    const result = await craftTransaction(
+      { address: tz2Address },
+      {
+        type: "send",
+        recipient: "tz2VYbGhf44HDDYP2fepNA7n7MDHWuBe6RxD",
+        amount: BigInt(1000000),
+        fee: { fees: BigInt(500).toString(), gasLimit: "10000", storageLimit: "0" },
```

Comment:
```suggestion
        fee: { fees: "500", gasLimit: "10000", storageLimit: "0" },
```
For homogeneity with `gasLimit`